source/clear-linux/get-started/virtual-machine-install/virtualbox-cl-installer.rst

```diff
@@ -3,256 +3,306 @@
 
+#. Select :guilabel:`OK`.
 
 Install |CL| on the |VB| VM
 ***************************
 
 |CL| is ready to be installed.
 
-
 Mount the installation ISO
 ==========================
-At this point, the newly created VM has a blank virtual hard disk with no 
-operating system.The |CL| installer ISO needs to be mounted as a virtual 
+
+At this point, the newly created VM has a blank virtual hard disk with no
+operating system. The |CL| installer ISO needs to be mounted as a virtual
 CD-ROM on the VM before powering the VM on.
 
-#. From the *VM - Settings* window, navigate to the *Storage* pane from the 
-   left-hand side.
+#. From the *ClearLinuxOs* :guilabel:`Settings` at left, select
+   :guilabel:`Storage`.
 
-#. From the middle *Storage Devices* column, click the blue CD disk labeled 
-   *Empty* under the *Controller: IDE* from.
+#. From :guilabel:`Storage Devices` in middle column, click the blue
+   disk labeled *Empty*.
 
-#. From the right-hand *Attributes* column, click the blue CD disk next to 
-   the *Optical Drive* drop down menu and click *Choose Virtual Optical 
+#. From the :guilabel:`Attributes` menu, click the blue CD disk next to
+   the *Optical Drive* drop down menu and click *Choose Virtual Optical
    Disk File...*
 
-   .. image:: ./figures/vbox/vbox-vm-settings-mount-ISO.png
-      :alt: Mounting an ISO in VirtualBox VM Settings  
+   .. figure:: figures/vbox/virtualbox-cl-installer-08.png
+      :scale: 100%
+      :alt: Choose Virtual Optical Disk Drive
+
+      Figure 8: Choose Virtual Optical Disk Drive
 
-#. A *choose a virtual optical disk file* browser window will appear. 
+#. A :guilabel:`choose a virtual optical disk file` browser window appears.
    Navigate to the extracted ISO file, select it, and click *Open*.
 
-   .. image:: ./figures/vbox/vbox-vm-settings-browse-ISO.png
-      :alt: Mounting an ISO in VirtualBox VM Settings  
-
-#. Click *OK* to exit the *VM Settings* menu and return to the main 
-   |VBM|.
+   .. figure:: figures/vbox/virtualbox-cl-installer-09.png
+      :scale: 100%
+      :alt: Mounting an ISO
 
+      Figure 9: Mounting an ISO
 
+#. Select :guilabel:`OK` to exit and return to the main |VBM|.
 
 Install |CL| using the installer
 ================================
 
-#. Start the VM from the |VBM| by selecting the |CL| VM and 
-   clicking *Start*
+#. In the |VBM|, select virtual machine you created, `ClearLinuxOS`,
+   and click *Start*
+
+   .. figure:: figures/vbox/virtualbox-cl-installer-10.png
+      :scale: 100%
+      :alt: Start the installer
+
+      Figure 10: Start the installer
 
-   .. image:: ./figures/vbox/vbox-start-vm.png
-      :alt: Starting a VirtualBox VM
+#. When the boot manager appears, select :guilabel:`Clear Linux Installer`
+   and select :guilabel:`Enter`.
 
-#. A new window of the VM console will appear and boot into the |CL|
-   installer. Follow the steps in :ref:`bare-metal-install-desktop` to
+#. Follow the steps in :ref:`bare-metal-install-server` to
```

Comment:
You've mixed the two methods here.  If I boot the live-desktop boot manager shows "Clear Linux* OS" without the word installer.  Both live-images will run the compatabilty test.  The desktop will boot into the gnome desktop, the server will say:
Welcome to the Clear Linux* OS live image!
blah, blah, blah
To configure the network run:
  nmtui
To install Clear Linux*OS onto this system please login as root,
enter a new temporary password, and run:
  clr-installer
clr-live login: